schemas/dublin-core/pom.xml

```diff
@@ -46,6 +46,26 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>sample-data</id>
+            <phase>compile</phase>
+            <configuration>
+<target>
+   <mkdir dir="target/sample-data"/>
+   <defaultexcludes add="**/.DS_Store"/>
+   <zip destfile="target/sample-data/natural_polar_ecosystems.mef"
+     basedir="src/main/plugin/dublin-core/sample-data/natural_polar_ecosystems"
+     excludes="**/.keepme"/>
+</target>
```

Comment:
```suggestion
              <target>
                <mkdir dir="target/sample-data"/>
                <defaultexcludes add="**/.DS_Store"/>
                <zip destfile="target/sample-data/natural_polar_ecosystems.mef"
                   basedir="src/main/plugin/dublin-core/sample-data/natural_polar_ecosystems"
                   excludes="**/.keepme"/>
              </target>
```

Reply:
I don't see any reason to be done on purpose, but if there is any reason for this @jodygarnett can clarify it.

Reply:
I was outdenting the ant `<target>` to make it very distinct from the surrounding `pom.xml` maven file.
I chose treat it like embedded script, rather than make a separate build.xml file.

I do not have my heart set on this change; it depends if other people find it clear?

Reply:
Generally, I would only do what you are trying to do if the contents we going to be appended or merged into another file where the indentation would look odd.  I'm not sure if I see the value in indenting the XML just to make it easier to see the ant block.  If this was the case then I think I could use that argument for a lot of xml blocks.

If this content is not being dumped to a file somewhere where the indentation will look weird then I personally prefer @josegar74 changes otherwise any editor that tries to format the xml will end up applying the standard indenting changes to that block as well and then it can be an issue to manage.  So generally, I prefer to follow the standards unless there is a good reason not to.

You could probably identify the blocks clearly using XML comments to identify the starting and ending block.

I also did a quick look the internet and could not find any existing code that apply this kind of indentation for the and plugin (not saying they don't exists)